consensus/wal_test.go

```diff
@@ -55,7 +55,7 @@ func TestWALTruncate(t *testing.T) {
 	// 60 block's size nearly 70K, greater than group's headBuf size(4096 * 10),
 	// when headBuf is full, truncate content will Flush to the file. at this
 	// time, RotateFile is called, truncate content exist in each file.
-	err = WALGenerateNBlocks(t, wal.Group(), 60)
+	err = WALGenerateNBlocks(t, wal.(*baseWAL).Group(), 60)
```

Comment:
now your previous solution somehow seems more appealing

Reply:
Yeah - it's generally a code smell when tests need to access private methods, but I'd rather not rewrite the tests or change the interface too much right now.

Reply:
Revert to exposing `BaseWAL`?

Reply:
Sure, done